recipes/simplification/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "simplification" %}
+{% set version = "0.7.10" %}
+{% set rdp_version = "0.13.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/simplification-{{ version }}.tar.gz
+  sha256: 4fbecf738eb36b6fe27c62824eb1c4d4f2700b56375e492fe67c1095747d07d3
+
+build:
+  skip: true  # [py<=37]
+  number: 0
+  script_env:
+    - RDPTAG={{ rdp_version }}
+  missing_dso_whitelist:  # [win]
+    - "*/rdp.dll"  # [win]
```

Comment:
Where is this coming from?

Reply:
The windows build, for some reason, during packaging (not building) cannot find this file, so I added it to the whitelist.